grr/server/grr_response_server/gui/ui/components/client_details/client_diff.ts

```diff
@@ -202,3 +202,59 @@ export function getClientVersions(clientSnapshots: Client[]):
 
   return clientChanges;
 }
+
+function getFirstStringsJoinedPath(path: any[]): string {
+  let tokens: string[] = [];
+  path.forEach(val => {
+    if (typeof val !== 'string') return;
+    tokens.push(val);
+  });
+
+  return tokens.join('.');
+}
+
+function getPathsOfChangedEntries(differences: Diff<Client, Client>[]):
+    string[] {
+  const pathsSetOfChangedEntries: Set<string> = new Set();
```

Comment:
This is a complex name that also contains the type (set), which is redundant. Why not use `changedPaths`?